guides/organizations.md

```diff
@@ -1,97 +0,0 @@
-# Organizations
```

Comment:
Should we have this guide redirect instead of deleting it?

Reply:
Good catch, intended to do this but forgot!